opba-db/README.md

```diff
@@ -0,0 +1,18 @@
+# Database population with production data
+
+## Populate database with data from CSV file
+
+To facilitate database population with bank data, `BankProtocolActionsSqlGeneratorTest.java` class contains a script that 
+generates all neccassary data, that can be used by liquibase plugin. The instruction how to use this script is the following:
+
+* add or replace your bank data to the `/Users/ivaniuminov/Workspace/open-banking-gateway/opba-db/src/main/resources/migration/migrations/banks.csv` 
```

Comment:
Please make a path relative:
`/Users/ivaniuminov/Workspace/open-banking-gateway/opba-db/src/main/resources/migration/migrations/banks.csv` 
to
`./opba-db/src/main/resources/migration/migrations/banks.csv`

Reply:
Done, thanks